### DISCUSSION TITLE
plays / collected / watcher via API

### DISCUSSION THREAD

**Discussion by linaspurinis** [trakt/trakt-api#312] [Q&A] [answered]
API call: **https://api.trakt.tv/movies/watched/period** returns "watcher_count", "play_count", "collected_count".
However, API call **https://api.trakt.tv/movies/id** does not return this info even with extended=full. 

Any way to get this info for specific movie/show?

Thank you.

**Comment by rectifyer** [comment, marked as answer]
https://trakt.docs.apiary.io/#reference/movies/stats/get-movie-stats

Sounds like that's the API method you're looking for.
